test/unit/services/TriggerService.test.js

```diff
@@ -118,11 +126,15 @@ describe('The TriggerService', function () {
             try {
                 await TriggerService.validateFieldUsingRegex(oid, record, options);
                 expect.fail("Should have thrown error");
-            } catch (err) {
-                expect(err).to.be.an('error');
-                expect(err.name).to.eq("RBValidationError");
-                expect(err.message).to.eq("Title is required Submission format is invalid");
-            }
+                        } catch (err) {
+                                expect(err).to.be.an('error');
+                                expect(err.name).to.eq("RBValidationError");
+                                const acceptable = [
```

Comment:
Why are the translations sometimes available and sometimes not? Can this test be made deterministic?